content/en/aws/managed-streaming-for-kafka/index.md

```diff
@@ -66,7 +68,155 @@ The expected output is something like the following
         },
         "NumberOfBrokerNodes": 3,
         "State": "ACTIVE",
-        "ZookeeperConnectString": "localhost:4515"
+        "ZookeeperConnectString": "localhost:4513"
     }
 }
-{{< / command >}}
\ No newline at end of file
+{{< / command >}}
+
+## Create a kafka topic
+
+In this step of using LocalStack MSK, you create a client machine. You use this client machine to create a topic that produces and consumes data.
+
+First, Install Java on the client machine by running the following command:
```

Comment:
I think we can remove this, and mention that Java is a prerequisite for this sample (also, not all users may be on Linux or have `yum` installed).

Reply:
Done